pages/jedi/meet.js

```diff
@@ -1,7 +1,14 @@
+import {
+  faDiscord,
+  faSlack,
+} from '@fortawesome/free-brands-svg-icons';
```

Comment:
Have we ported over all the other icons we use in the website to use font awesome?

Reply:
Yup!